UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Utilities/FileUtility.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography;
+using DiscUtils.Iso9660;
+namespace UKHO.PeriodicOutputService.Common.Utility
```

Comment:
Namespace shoud be UKHO.PeriodicOutputService.Common.**Utilities** to correspons to the file location.

Reply:
Renamed